rct229/ruleset_functions/get_surface_conditioning_category_dict.py

```diff
@@ -1,102 +1,104 @@
 import pandas as pd
 
 from rct229.ruleset_functions.get_zone_conditioning_category_dict import (
-    CONDITIONED_MIXED,
-    CONDITIONED_NON_RESIDENTIAL,
-    CONDITIONED_RESIDENTIAL,
-    SEMI_HEATED,
-    UNCONDITIONED,
-    UNENCLOSED,
+    ZoneConditioningCategory,
     get_zone_conditioning_category_dict,
 )
 from rct229.utils.jsonpath_utils import find_all
 
 # Constants
 # TODO: These should directly from the enumerations
-EXTERIOR = "EXTERIOR"
-GROUND = "GROUND"
-INTERIOR = "INTERIOR"
-# Surface conditioning categories (export these)
-EXTERIOR_MIXED = "EXTERIOR MIXED"
-EXTERIOR_NON_RESIDENTIAL = "EXTERIOR NON-RESIDENTIAL"
-EXTERIOR_RESIDENTIAL = "EXTERIOR RESIDENTIAL"
-SEMI_EXTERIOR = "SEMI-EXTERIOR"
-UNREGULATED = "UNREGULATED"
+
+
+# Intended for export and internal use
+class SurfaceConditioningCategory:
+    """Enumeration class for zone conditioning categories"""
+
+    EXTERIOR: str = "EXTERIOR"
```

Comment:
I suggest not including EXTERIOR, GROUND, INTERIOR here because these are not surface conditioning categories.  Ideally, we would get these values from the schema enumerations.  In the mean time, they can be constants in this file outside of the SurfaceConditioningCategory class.

Reply:
To import the enum, will this looks good?
`SurfaceAdjacentTo = schema_enums["SurfaceAdjacentTo"]`
`                if surface_adjacent_to == SurfaceAdjacentTo.INTERIOR.name
                else surface_adjacent_to,`

Reply:
That works.  At some point, we should try to find a more elegant syntax for handling the enumerations. The `.name` part is just ugly. Probably, instead of using Python's Enumeration type, we should just create a class for each enumeration. Then the syntax could simply be `SurfaceAdjacentTo.INTERIOR` without the `.name`.

I think all we will need to do is to replace line 38 of rct229/data/schema_enums.py with:
```
schema_enums = {
    class_name: type(class_name, (object, ),  enum_dict) 
    for class_name, enum_dict in _enum_dicts.items()}
```
The `type(class_name, (object, ),  enum_dict)` part creates a new type (class) with the given class name, that inherits from `object`, and has class variables given by `enum_dict`.

Let me know if you'd like me to make this change and update wherever schema_enums is used.

Reply:
Yea, could you update the method? I agree the .name is ugly and the functioin in schma_enums should takes care of the .name. Thanks!